parsl/executors/radical/executor.py

```diff
@@ -400,8 +400,11 @@ def task_translate(self, tid, func, parsl_resource_specification, args, kwargs):
 
     def _pack_and_apply_message(self, func, args, kwargs):
         try:
-            buffer = pack_apply_message(func, args, kwargs,
-                                        buffer_threshold=1024 * 1024)
+            buffer = pack_res_spec_apply_message(func,
```

Comment:
It's not clear to me why this doesn't work with the earlier pack_apply_message implementation - the other end is still using the original unpack_apply_message, right?

Reply:
Radical uses the `execute_task` from the HTEX process_worker_pool which now uses the updated `unpack_res_spec_apply_message` method. I updated both the worker side calls and the executor to match this.